### PR TITLE
cpdf: add Spanish translation

### PR DESCRIPTION
**Pull request by ivanhercaz**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by ivanhercaz** [comment]
This PR should wait to solve the review made by @agnivade in #3713 and the  doubt I posted.

File: pages.es/common/cpdf.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> Interfaz de línea de comandos para manipular documentos PDF existentes de diferentes maneras.
+> Más información: <https://www.coherentpdf.com/cpdfmanudel/cpdfmanudel.html>.
+
+- Selecciona las páginas 1, 2, 3 y 6 del documento fuente y escribirlas en el documento objetivo:
+
+`cpdf {{ruta/del/documento_fuente.pdf}} 1-3,6 -o {{ruta/del/documento_objetivo.pdf}}`
```
Comment (zdroid) [inline pages.es/common/cpdf.md]: 1-3, 6 should be in tokens.

Comment (sbrl) [reply pages.es/common/cpdf.md]:
```suggestion
`cpdf {{ruta/del/documento_fuente.pdf}} {{1-3,6}} -o {{ruta/del/documento_objetivo.pdf}}`
```

Comment (ivanhercaz) [reply pages.es/common/cpdf.md]: Fixed @zdroid, @sbrl!

**P.S.** Happy new year!

Comment (zdroid) [reply pages.es/common/cpdf.md]: Feliz año nuevo! :fireworks:

**Pull request merged**